mods/ra/maps/allies-02/allies02.lua

```diff
@@ -87,6 +89,14 @@ RunInitialActivities = function()
 		SendTrucks()
 	end)
 
+	Trigger.OnAllKilled(SovietBase, function()
+		Utils.Do(ussr.GetGroundAttackers(), function(unit)
```

Comment:
Idea for future improvement: Add a `Utils.Except` helper, so this could simplify to
```lua
Utils.Do(Utils.Except(ussr.GetGroundAttackers(), PathGuards), function(unit)
    Trigger.OnIdle(unit, unit.Hunt)
end)
```